client/src/components/Filters/index.js

```diff
@@ -50,6 +50,7 @@ class Filters extends Component {
         Header: 'Rules count',
         accessor: 'rulesCount',
         className: 'text-center',
+        Cell: props => props.value.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ','),
```

Comment:
Using a regular expression is a bit too much for that:)

Use a [culture-specific approach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) instead: `new Number(100000).toLocaleString()`

Reply:
Thanks for the suggestion and it's great. I've pushed another commit.